autoPyTorch/api/base_task.py

```diff
@@ -1180,6 +1169,95 @@ def fit(self,
         self._clean_logger()
         return pipeline
 
+    def fit_ensemble(
```

Comment:
Is this the post-hoc ensemble fitting?
It is confusing for me to have `_fit_ensemble` and `fit_ensemble`.

Reply:
yeah let me fix the documentation for this, sorry. But basically, `_fit_ensemble` is a private method which helps to not repeat the same code as it is being done in the`search` method already. I'll change the name for this method to also clear the confusion